airflow-comparison.md

```diff
@@ -0,0 +1,47 @@
+## Lineage
+
+Data lineage broadly refers to a collection of tools, practices and approaches which allow the tracking of the flow of data within a data pipeline. Data lineage tools maintain a record of data within its lifecycle, including its source and any transformations applied to it. The main purpose of data lineage is to provide a platform for ensuring data quality, as well as a way of troubleshooting any errors which arise within the data.
```

Comment:
> Data lineage tools maintain a record of data within its lifecycle, including its source and any transformations applied to it.

Ideally, data lineage capability e2e test would be data transformation flow replay, I think. If we agree, we may consider additionally elaborating the vdk-lineage README.md. 

Yet the current VDK lineage state of the art described in it:
```
At POC level currently.
Currently, lineage data is collected:
* For each data job run/execution both start and end events including the status of the job (failed/succeeded)
* For each execute query we collect input and output tables.
```